src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,15 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String bladeType;
+
     public Bulldozer() {
     }
 
+    public void setBladeType(String bladeType) {
+        this.bladeType = bladeType;
+    }
+
```

Comment:
Recommend to create getters and for other similar classes too.

Reply:
They didn't say "always create getters". They said "You should never want to expose the object fields directly. They should be accessed through special methods", so if you need read it, you should create getter. But i don't